tests/charts/test_dag_processor.py

```diff
@@ -549,3 +550,8 @@ def test_should_add_component_specific_annotations(self):
         )
         assert "annotations" in jmespath.search("metadata", docs[0])
         assert jmespath.search("metadata.annotations", docs[0])["test_annotation"] == "test_annotation_value"
+
+
+class TestDagProcessorLogGroomer(LogGroomerTestBase):
+    obj_name = "dag-processor"
+    folder = "dag-processor"
```

Comment:
This will likely fail static checks

Reply:
Thanks for the check!
Added newline at the end of the code.